pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -209,6 +211,20 @@ func Train(trainIR *ir.TrainClause) (string, error) {
 			fmt.Sprintf("\"%s\": [%s]", target, strings.Join(perTargetFeatureColumnsCode, ",\n")))
 	}
 	isKeras, estimatorStr := IsKerasModel(trainIR.Estimator)
+	isPAI := os.Getenv("SQLFLOW_submitter") == "pai"
+	paiTable := ""
+	// TODO(typhoonzero): if isPAI, create two Couler steps
+	if isPAI {
+		fromRegex, err := regexp.Compile("FROM\\s([a-z0-9_\\.]*)")
```

Comment:
1. Use a raw string to get rid of escaping?
1. Use `\s*` in case there is more than one whitespace, and I think we may need a multiline setting to make the following work on this.
    ```sql
    select * from 
        iris.trainable ...
    ```
3. May the table name capitalized? If so, we'd better use `[\w\.]` instead of `[a-z0-9_\\.]`